lib/mayaHydra/hydraExtensions/adapters/lightAdapter.cpp

```diff
@@ -154,12 +154,7 @@ VtValue MayaHydraLightAdapter::Get(const TfToken& key)
         MFnLight       mayaLight(GetDagPath());
         GlfSimpleLight light;
         const auto     color = mayaLight.color();
-        auto     intensity   = mayaLight.intensity();
```

Comment:
You probably don't have the up to date file as this shouldn't be removed, unless you moved it somewhere else ?

Reply:
yes, need merge with latest

Reply:
Sounds good. Thanks.